wrt/wrt-uxmanu-android-tests/wrt-ux-app/testscripts/Crosswalk_WebApp_HTML5_Notification.feature

```diff
@@ -0,0 +1,7 @@
+Feature: wrt-ux-app
+ Scenario: Crosswalk WebApp HTML5 Notification
+  When launch "web_feature_notification_tests"
+    And I click button "Get Notification"
+    And I open notification
+   Then I should see text "New Email Received"
```

Comment:
This is just a workaround, do not resolve the problem. Let's find out a step to save the text what you intend to see.